src/AddResident.js

```diff
@@ -298,9 +305,9 @@ class AddResidentForm extends Component {
                             options={this.state.villageOptions}/>
                     <label style={{paddingRight: "0px", paddingLeft: "10px"}}>House Number:</label>
                     {this.state.village !== "" ?
-                      <Select style={{width: "200px", margin: "10px"}} value={this.state.house} id="getHouse" className="dropdown" onChange={(option) => {this.handleDropdownMulti(option, "house")}} name="house"
-                      options={this.state.villageHouses[this.state.village.value]}/> : <Select style={{width: "200px", margin: "10px"}} value={this.state.house} id="getHouse" className="dropdown" onChange={(option) => {this.handleDropdownMulti(option, "house")}} name="house"
-                              options={[]}/>}
+                        <Select style={{width: "200px", margin: "10px"}} value={this.state.house} id="getHouse" className="dropdown" onChange={(option) => {this.handleDropdownMulti(option, "house")}} name="house"
+                                options={this.state.villageHouses[this.state.village.value]}/> : <Select style={{width: "200px", margin: "10px"}} value={this.state.house} id="getHouse" className="dropdown" onChange={(option) => {this.handleDropdownMulti(option, "house")}} name="house"
+                                                                                                         options={[]}/>}
```

Comment:
Can this ternary not be pushed to just the options variable?
The Select tag is the same except for the options variable.